### PR TITLE
Mag-Gripper Light Tiles

### LINKED ISSUE #3216: Mag Gripper + Light Tiles
Magnetic Grippers cannot pick up Light Tiles, thus preventing maintenance drones from providing nice, glowy floors with colors that indicate departments or what-have-you.

Clearly, this is a top priority issue, and should be fixed with extreme prejudice.

### PR DESCRIPTION
**Pull request by Deanthelis**
Simply put, resolves #3216 by adding light tiles to the gripper. Drones can now improve the station by creating and placing light tiles.

This PR is mostly practice for me to get used to proper PR procedure with changelogs, etc., though clearly, the issue resolved is one of vital and immediate importance.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
3 commits->1 line change.

You need to update, in the command line, and not the web interface, before committing things to PRs--you shoudl never have merge commits in a newly submitted PR.

Instructions on github for windows: https://help.github.com/articles/syncing-a-fork/

**Comment by Deanthelis** [comment]
I made an entirely new fork and only made a single commit in bash, I don't know why it included those.

Git frustrates me.

**Comment by Crazylemon64** [comment]
`git rebase -i HEAD~3`, then squash the bottom two commits

**Comment by Fox-McCloud** [comment]
@Deanthelis 

Squash dem commits.

**Comment by Deanthelis** [comment]
Well, I did git rebase -i HEAD~3, and git proceeded to give me a list of every single commit in the entire repository. What's odd is only the last one is mine, 

http://puu.sh/mxuFm.png

I have no idea what I'm doing wrong, as usual with Git.

**Comment by tigercat2000** [comment]
@Crazylemon64 @Fox-McCloud Merge commits, makes it very difficult to rebase. Try it, I never figured it out.
@Deanthelis I would recommend running the following (minus the `(comment)`'s)
```
git reset --soft HEAD~1        (soft reset, destroy HEAD~1 (last) commit, leave changes in working tree)
git stash                      (stashes the working tree data)
git fetch upstream             (fetches latest code from 'upstream' which should be this repo)
git reset --hard upstream/master                      (hard-reset, erases working tree)
git stash list                                        (view a list of created stashes)
git stash apply stash@{number from above stash}       (it will probably be stash@{0})
git stash drop stash@{same number}                    (probably 0, drops the stash from the saves)
```

**Comment by tigercat2000** [comment]
and if that goes horribly wrong and you need to just abandon it, just run
```
git reset --hard origin/DroneLightTiles
```
to reset to the PR data

**Comment by Deanthelis** [comment]
I did all of those things, and it seems to have worked properly, but it fails to push because my local files are now apparently behind what Github has stored.

Should I just push to a new branch and obliterate the old one?

**Comment by tigercat2000** [comment]
Ah. Yeah, any sort of rebasing makes git yell at you when trying to push, it hates history edits. You can overwrite branch data by including `-f` in the push command (`git push origin/DroneLightTiles -f`)

**Comment by Deanthelis** [comment]
All hail @tigercat2000 , savior of maintenance drones and hand-holder of people terrible at git. That appears to have worked.

**Comment by tigercat2000** [comment]
there is probably a proper way to do that, but it's better than my typical "fuck it remake half the code manually on a second branch and overwrite the first"

**Comment by Deanthelis** [comment]
Oh dear.
It still thinks I'm in the middle of rebasing, I can't checkout to other branches now for some reason.

You're my only hope, Tiger-wan 2000.

**Comment by tigercat2000** [comment]
@Deanthelis run `git rebase --abort`

**Comment by Deanthelis** [comment]
I tried reading some of the documentation for this crap, but none of it made sense because it seemed to assume I already knew what the hell all the terms meant. I probably caused most of what was wrong with this PR by trying things I really didn't understand.

tl;dr git is literally magic

**Comment by tigercat2000** [comment]
i've been playing with the shit for almost a year and I still don't understand it
